### PR TITLE
Update GetNumberOfColumnsFromRange in order to support columns after AA

### PR DESCRIPTION
**Pull request by DanielaDumitrascu**
Issue Description: Read Range with range columns after AA return "Read range : Input array is longer than the number of columns in this table". 
The changes I made seem to work, but should be also tested more and it needs to be improved  (like the name of the columns in the range could be lower case also? if so, the regular expression be updated)

### CODE REVIEW

File: Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs [hunk 1]
```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
...
             }
 
-            return secondLetter - firstLetter + 1;
+            return secondColNumber - firstColNumber + 1;
+
+            //return secondLetter - firstLetter + 1;
         }
 
+
+        public static int letterToColumn(String letter)
```
Comment (bogdan-popescu-uipath) [inline Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs]: Please make sure that current unit tests are still passing and add new ones for the added functionality.

File: Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs [hunk 2]
```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
...
 
-            if (secondLetter < firstLetter)
+            /* if (secondLetter < firstLetter)
+             {
+                 throw new Exception("Invalid range specified.");
+             }*/
+
+
+            // Handle also columns after AA
+            Regex regexRange = new Regex(@"([A-Z]+)\d+");
```
Comment (adrian-zanescu) [inline Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs]: We could initialize the regex as a static member and compile it to reuse it between calls

File: Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs (hunk 1 above)
Comment (adrian-zanescu) [inline Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs]: maybe LetterToColumn for C# consistency

File: Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs [hunk 3]
```diff
@@ -94,14 +95,58 @@ public static int GetNumberOfColumnsFromRange(string range)
...
+            string firstColName, secondColName;
+            int firstColNumber = 0, secondColNumber = 0;
+            if (match.Success)
+            {
+                firstColName = match.Groups[1].Value;
+                firstColNumber = letterToColumn(firstColName);
+            }
+            else
+            {
+                throw new Exception("Invalid range specified.");
```
Comment (adrian-zanescu) [inline Google Spreadsheet/GoogleSpreadsheet.Activities/ReadRange.cs]: We should try to throw more specific exception types like ArgumentException or InvalidOperation

**Pull request merged**